collect_app/src/main/java/org/odk/collect/android/utilities/FormEntryPromptUtils.java

```diff
@@ -141,7 +141,7 @@ public static String getBindAttribute(FormEntryPrompt prompt, String attributeNa
     }
 
     @Nullable
-    public static String getAttribute(FormEntryPrompt formEntryPrompt, String attributeName) {
+    public static String getBodyAttribute(FormEntryPrompt formEntryPrompt, String attributeName) {
         String acc = formEntryPrompt.getQuestion().getAdditionalAttribute(null, attributeName);
```

Comment:
World's tiniest nit: `acc` doesn't make any sense anymore (it used to mean accuracy).